Header       header

BoundingBox  boundingBox
float32        propability
uint32       objectId